Package.swift

```diff
@@ -3,7 +3,8 @@ import CompilerPluginSupport
 import PackageDescription
 
 let swiftFeatures: [SwiftSetting] = [
-    .enableUpcomingFeature("ExistentialAny")
+    .enableUpcomingFeature("ExistentialAny"),
+    .unsafeFlags(["-cross-module-optimization"])
```

Comment:
Does using unsafe flags affect warnings end users see when integrating SwiftLint as a SwiftPM plugin?

Reply:
As per the [documentation](https://developer.apple.com/documentation/packagedescription/swiftsetting/unsafeflags(_:_:)#discussion), it does cause errors for version-based dependencies. While that might be okay on macOS due to the use of `SwiftLintBinary`, it's an issue on Linux where everything has to be built.